Fixing bugs, adding title which is missing because of lack of nav (#1724)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/sign-into-web-app-remediation/handle-remediation/index.md

Next you will need to handle the back-and-forth remediation process between your client and the Okta Identity Engine.

> The end result of this will be an interaction code, that in the <GuideLink link="../get-tokens">next step</GuideLink> will be exchanged for access and ID tokens.

Reviewer: ```suggestion
The end result of this process is an interaction code, which in the <GuideLink link="../get-tokens">next step</GuideLink> is exchanged for access and ID tokens.
```
